### PR TITLE
chore: update readme status for November

### PR DESCRIPTION
**Pull request by jelbourn**
R: @kara

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,14 +65,14 @@ High level items planned for October 2016:
 | dialog           |  Started, not yet ready for release | [README][22] |   [#114][0114] |
 | snackbar / toast | Initial version, needs enhancements | [README][21] |   [#115][0115] |
 | select           |                      Design started |           -  |   [#118][0118] |
```
Comment (kara) [inline README.md]: Do we want to put `select` as started? Given that the first PR is in master.

**Comment by fxck** [comment]
Could you include some info about releases? I could really use some of the stuff/fixes that are in master..

**Comment by jelbourn** [comment]
@fxck we're doing an alpha.10 tomorrow unless something goes awry

**Comment by jelbourn** [comment]
@fxck some overlay and menu changes collided, causing bugs with menu that need to be resolved before a release. It will probably be Monday now.

**Pull request merged**